### PR TITLE
update banner'

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

Fest happening as part of RH Summit
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs/docsite/.templates/banner.html

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by samccann** [comment]
image

File: docs/docsite/.templates/banner.html [hunk 1]
```diff
@@ -20,7 +20,10 @@
       '<div id="latest_extra_banner_id" class="admonition important">' +
       '<br>' +
       '<p>' +
-        'Continue building on your automation knowledge, visit the <a href="https://events.experiences.redhat.com/widget/redhat/rhaf22/SessionCatalog2022">AnsibleFest content hub!</a>  ' +
+        '<b>Experience AnsibleFest at Red Hat Summit</b>' +
+      '</p>' +
+      '<p>' +
+        'We\'re bringing AnsibleFest to Red Hat Summit in Boston, May 23-25. It\'s the best of Fest―combined with Red Hat Summit for even more learning and technical engagement. <a href="https://www.redhat.com/en/summit/ansiblefest?intcmp=7013a0000034lvhAAA">Learn more.</a>  ' +
```
Comment (oraNod) [inline docs/docsite/.templates/banner.html]:
```suggestion
        'We\'re bringing AnsibleFest to Red Hat Summit in Boston, May 23-25. It\'s the best of Fest combined with Red Hat Summit for even more learning and technical engagement. <a href="https://www.redhat.com/en/summit/ansiblefest?intcmp=7013a0000034lvhAAA">Learn more.</a>  ' +
```

Comment (oraNod) [reply docs/docsite/.templates/banner.html]: not sure why that is hyphenated but it looks weird to me

Comment (samccann) [reply docs/docsite/.templates/banner.html]: well it was an em-dash in the original request but yeah, happy to take the suggested edit.

**Review by oraNod — APPROVED** [review]

**Pull request merged**